tasks/actorstate/miner.go

```diff
@@ -143,13 +154,13 @@ type MinerStateExtractionContext struct {
 	CurrTs    *types.TipSet
 }
 
-func (m *MinerStateExtractionContext) IsGenesis() bool {
-	return m.CurrTs.Height() == 0
+func (m *MinerStateExtractionContext) NoPreviousState() bool {
```

Comment:
Can we invert the logic here and rename to `HasPreviousState`

It makes it much easier to follow the logic in `ExtractMinerInfo` where we have a lot of `if !ec.NoPreviousState()` (makes me mentally have to parse "if not no previous state"). I'd rather read that as `if ec.HasPreviousState()`